packages/dapp/src/reducers/index.ts

```diff
@@ -117,5 +124,17 @@ export default combineReducers({
   challengeUserData,
   appealChallengeUserData,
   government,
+});
+
+const networkDependent = (state: any, action: AnyAction) => {
+  if (action.type === networkActions.SET_NETWORK) {
+    return networkDependentReducers(undefined, action);
```

Comment:
why does this need undefined passed in

Reply:
that's what the example I looked up said to do to reset state